locales/hi/php.json

```diff
@@ -61,7 +61,7 @@
     "lte.file": "The :attribute must be less than or equal :value kilobytes.",
     "lte.numeric": "The :attribute must be less than or equal :value.",
     "lte.string": "The :attribute must be less than or equal :value characters.",
-    "mac_address": "The :attribute must be a valid MAC address.",
+    "mac_address": ":Attribute एक मान्य MAC address होना चाहिए.",
```

Comment:
Here, the application should definitely end with a full stop, and not the `।` character?